plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -932,6 +945,44 @@
               radio_band: "6 GHz"
               channel_width: "40 MHz"
       register: output_list
+
+    - name: Reboot single or multiple access point
+      cisco.dnac.accesspoint_workflow_manager:
+        dnac_host: "{{ dnac_host }}"
+        dnac_username: "{{ dnac_username }}"
+        dnac_password: "{{ dnac_password }}"
+        dnac_verify: "{{ dnac_verify }}"
+        dnac_port: "{{ dnac_port }}"
+        dnac_version: "{{ dnac_version }}"
+        dnac_debug: "{{ dnac_debug }}"
+        dnac_log: True
+        dnac_log_level: DEBUG
+        config_verify: True
+        state: merged
+        config:
+          - reboot_ap:
+            - mac_address: 6c:d6:e3:75:5a:e0
+            - mac_address: e4:38:7e:42:bc:00
+      register: output_list
+
+    - name: Reboot single or multiple access point by hostname
+      cisco.dnac.accesspoint_workflow_manager:
+        dnac_host: "{{ dnac_host }}"
+        dnac_username: "{{ dnac_username }}"
+        dnac_password: "{{ dnac_password }}"
+        dnac_verify: "{{ dnac_verify }}"
+        dnac_port: "{{ dnac_port }}"
+        dnac_version: "{{ dnac_version }}"
+        dnac_debug: "{{ dnac_debug }}"
+        dnac_log: True
+        dnac_log_level: DEBUG
+        config_verify: True
+        state: merged
+        config:
+          - reboot_ap:
```

Comment:
-  reboot_aps:
        mac_addresses:
          - "AA:BB:CC:DD:EE:FF"
          - "11:22:33:44:55:66"

Reply:
Updated and changed the code according to this input.